.github/workflows/pr.yml

```diff
@@ -0,0 +1,113 @@
+name: Master - PR
```

Comment:
Since there are triggers for both push and PR, maybe call this CI instead of PR.